website/docs/wallet/withdraw-validator.md

```diff
@@ -347,37 +322,37 @@ and you should see a response that contains withdrawal credentials that should h
 </TabItem>
 <TabItem value="full">
 
-## Step-by-Step Guide: Full Withdrawals
+## Option 2: Full withdrawals
 
 :::caution
-⚠️ **DO NOT ATTEMPT TO CHANGE YOUR BLS KEYS ON MAINNET** until the client software is ready to accept your BLS key changes. This is feature is not yet available on mainnet at the time of writing on February 1st, 2023
+**DO NOT ATTEMPT TO CHANGE YOUR BLS KEYS ON MAINNET** until the client software is ready to accept your BLS key changes. This is feature is not yet available on mainnet at the time of writing on February 1st, 2023
 :::
 
+To fully withdraw a validator and its earnings, your validator needs to also be exited in addition to having its [withdrawal credentials](https://github.com/ethereum/consensus-specs/blob/master/specs/phase0/validator.md#withdrawal-credentials) changed.
 
-To fully withdraw a validator and its earnings the validator needs to also be exited in addition to having its [withdrawal credentials](https://github.com/ethereum/consensus-specs/blob/master/specs/phase0/validator.md#withdrawal-credentials) changed.
+Please follow our [exiting-a-validator how-to](exiting-a-validator.md).
 
-Please follow the guide [exiting-a-validator](exiting-a-validator.md).
-
-Please follow the partial withdrawal guide to change the validator's withdrawal credentials.
+Refer to the above partial withdrawal guidance to change your validator's withdrawal credentials.
 
 :::caution
-⚠️ instructions for setting withdrawal address do not need to be repeated if withdrawal_credentials are updated to the `0x01` prefix.
-
+Instructions for setting your withdrawal address do not need to be repeated if withdrawal_credentials are updated to the `0x01` prefix.
 :::
 
-Once the validator is both exited as well as having its withdrawal credentials changed then the validator will automatically be withdrawn when a block proposer processes the withdrawal. **note:** a maximum of 16 withdrawals can be processed per block.
+Once the validator is both exited as well as having its withdrawal credentials changed, the validator will automatically be withdrawn when a block proposer processes the withdrawal. **Note that a maximum of 16 withdrawals can be processed per block.**
 
 </TabItem>
 </Tabs>
 
 
 ## Frequently asked questions
 
+<!-- TODO: These questions can now be moved into FAQ CMS and embedded both here and within our root-level FAQ - ping Mick if you'd like to help with this. -->
+
 **Q: When can I withdraw?**
 
 A: After the Capella/Shanghai hardfork withdrawals will be enabled. This is expected to go live this spring, 2023.
 
-**Q: I forgot my mnemonic what can I do?**
+**Q: I forgot my mnemonic, what can I do?**
```

Comment:
We can continue editing the FAQ section post-publish - with the public preview banner, there's no need to hold off on perfection. I'd recommend publishing this "way more than good enough" version sooner rather than later and iterating on it leading up to mainnet / based on feedback we receive.

Reply:
I totally agree 👍